_commands/tools/bzip2.md

```diff
@@ -0,0 +1,44 @@
+---
+---
+
+bzip2
+-------
+<!--TODO: Add documentation for this command by submitting a pull request.-->
+<!-- one line explanation would go here -->
+`bzip2` is the command used for compressing and decompressing files.
+
+<!-- minimal example -->
+~~~ bash
+$ bzip2 filename
+~~~
+
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `bzip2`
+
+~~~bash
+$ bzip2 [Option(s)] filename(s)
+~~~
+
+Options include:
+
+1. `-v` for verbose: provide confirmation and degree to which the data is compressed
+2. `-s` for small: reduce the memory usage for compression
+3. `-d` for decompress: decompress the specified file
+
+----
+
+#### `bzip2 file1 file2 file3`
+
+~~~bash
+$ bzip2 file1 file2 file3
+~~~
+
+##### Break it down
```

Comment:
Remove this placeholder heading or replace it with a more meaningful heading.

Reply:
I'm confused. Do you mean "##### Break it down"?

Reply:
placeholder heading removed